scripts/do-release.sh

```diff
@@ -203,7 +203,11 @@ function update_operator_pr() {
         update_go_mod "${target}"
     done
 
-    sed -i -E "s/(.*Version +=) .*/\1 \"${release['version']#v}\"/" "projects/${project}/apis/submariner/v1alpha1/versions.go"
+    local versions_file
+    versions_file=$(grep -l -r 'Default.*Version *=' projects/${project}/ | grep '.go$')
+    [[ -n "${versions_file}" ]] || { printerr "Can't find file for default image versions"; return 1; }
+
+    sed -i -E "s/(.*Version +=) .*/\1 \"${release['version']#v}\"/" "${versions_file}"
```

Comment:
```suggestion
    sed -i -E "s/(Default.*Version *=) .*/\1 \"${release['version']#v}\"/" "${versions_file}"
```

to match the pattern used in the `grep` command, and ensure we avoid replacing any non-default versions (if `versions.go` is changed to include other versions).